oidc-api/src/main/java/uk/gov/di/authentication/oidc/lambda/LogoutHandler.java

```diff
@@ -97,25 +100,37 @@ public APIGatewayProxyResponseEvent logoutRequestHandler(APIGatewayProxyRequestE
                         () -> sessionService.getSessionFromSessionCookie(input.getHeaders()));
         attachSessionToLogsIfExists(sessionFromSessionCookie, input.getHeaders());
 
+        Optional<String> subjectId = Optional.empty();
+        Optional<String> sessionId = Optional.empty();
+        if (sessionFromSessionCookie.isPresent()) {
+            subjectId =
+                    Optional.ofNullable(
+                            sessionFromSessionCookie.get().getInternalCommonSubjectIdentifier());
+            sessionId = Optional.ofNullable(sessionFromSessionCookie.get().getSessionId());
+        }
```

Comment:
We can collapse these using `.map()` rather than testing if `sessionFromSessionCookie` exists directly.

```
var subjectId = sessionFromSessionCookie.map(Session::getInternalCommonSubjectIdentifier);
var sessionId = sessionFromSessionCookie.map(Session::getSessionId);
```